docs/development/release.md

```diff
@@ -281,7 +288,7 @@ If this is the latest version, you can simply create a PR to merge the minor ver
 
 Do not merge `master` back into the release branch since we want the release branch to _just_ have the code from the release.  Instead, create a new branch off of the release branch and merge that to master. You can freely merge master into this branch and delete it when it is merged to master.
 
-If this is a backport, you will need to create a PR that ports the changelog updates back to master. It is important in this changelog note to call it out as a backport. For example, fixes in backport version `1.14.5` may not be available in `1.15.0` because the fix was introduced in `1.15.3`.
+**Please Note** If this is a backport, you will need to create a PR that merges **ONLY** the changelog updates back to master. It is important in this changelog note to call it out as a backport. For example, fixes in backport version `1.14.5` may not be available in `1.15.0` because the fix was introduced in `1.15.3`.
```

Comment:
Looks good. 2 suggestions for small changes:
- add quotes ("") around the "fixes in backport version..." as that seems to be the actual language that is used in these PRs
- add instructions for adding release notes, since that seems to be missing. Something like: add release notes by copying changelog changes to the release, and see existing releases for how to format.

Reply:
Yup! I like it. Check out the latest diff. I also updated the changelog note to more closely match our more recent wording. let me know what you think.

Reply:
Looks good!